CHANGELOG.md

```diff
@@ -13,6 +13,27 @@ All notable changes to this project will be documented in this file.
 This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.38.0]
+
+### News around this release
+
+* [Phillip Chlap](https://github.com/pchlap) has joined as a pymedphys Maintainer
+* Thanks to [Cedric](https://github.com/cedricbelanger) for reporting and fixing DICOM coordinates bug: [Discourse](https://pymedphys.discourse.group/t/gamma-index-calculation-possible-issues/191/2)
+
+### Bug Fixes
+
+* DICOM Coordinates
+  * Fix pixel spacing bug [Pull Request](https://github.com/pymedphys/pymedphys/pull/1571)
+* Pinnacle Export Tool
+  * Write DICOM files to be more compatible with other software [Pull Request](https://github.com/pymedphys/pymedphys/pull/1578)
+* Daily Winston-lutz
+  * Small workaround [Pull Request](https://github.com/pymedphys/pymedphys/pull/1565)
+
+### Breaking Changes
+
+* Un-expose pseudonymise, susceptible to brute-force reversal [Pull Request](https://github.com/pymedphys/pymedphys/pull/1534)
+* Mitigated in [Pull Request](https://github.com/pymedphys/pymedphys/pull/1580), not yet re-exposed with the API
```

Comment:
@sjswerdloff would you prefer we hold the release back until the pseudonymise API can be re-exposed exactly as it was before?

This would actually result in all changes being bug fixes (no new features, no breaking changes). As a heads up to @pchlap that would make the release number be `0.37.2`.

Reply:
> @sjswerdloff would you prefer we hold the release back until the pseudonymise API can be re-exposed exactly as it was before?
> 
> This would actually result in all changes being bug fixes (no new features, no breaking changes). As a heads up to @pchlap that would make the release number be `0.37.2`.

I would, but is there something I need to change to re-expose the API? 
I'm having a tough time finding the changes that un-exposed it...

Reply:
> @sjswerdloff, just want to bring the following comment to your attention:
> 
> [#1601 (comment)](https://github.com/pymedphys/pymedphys/pull/1601#discussion_r770281096)

Yes, but is there a simple way to revert or merge back from PR 1534?

Reply:
Yup, need to reverse the following changes to the following files:

* lib/pymedphys/_streamlit/apps/__init__.py
* lib/pymedphys/cli/experimental/__init__.py
* lib/pymedphys/docs/cli/ref/index.rst
* lib/pymedphys/docs/lib/ref/experimental/index.rst
* lib/pymedphys/experimental/pseudonymisation.py
* lib/pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py

Within the following PR:

https://github.com/pymedphys/pymedphys/pull/1534/files#diff-34612890b5c93f3a57871e49573002e6a84361fc633d63760ccf6aaee36beac0

---

I might have missed a file or two...

Reply:
> Yes, but is there a simple way to revert or merge back from PR 1534?

I believe the following may be your friend:

https://www.atlassian.com/git/tutorials/undoing-changes/git-revert

Reply:
I would use it on the PR Merge commit:

https://github.com/pymedphys/pymedphys/commit/72f5e72316b7761527757b5a53fd7eeadd5dcdcb

Instead of trying to use it on each individual commit

Reply:
Resolved in https://github.com/pymedphys/pymedphys/pull/1613